Jenkinsfile

```diff
@@ -93,10 +93,12 @@ node('test') {
 
 def functionalDynamicParallelSteps(image){
     ciGroupsMap = [:]
-    for (int i = 1; i <= 3; i++) {
+    for (int i = 1; i <= 12; i++) {
         def currentCiGroup = "ciGroup${i}";
         def currentStep = i;
         ciGroupsMap["${currentCiGroup}"] = {
+            sh "rm -rf ${env.WORKSPACE}_${currentCiGroup}"
+            sh "mkdir ${env.WORKSPACE}_${currentCiGroup}"
```

Comment:
This is not a good design. Please take a look at the last part of my comment. https://github.com/Tengda-He/kibana/issues/57#issuecomment-982808830
We should not extent the execution to the outside of the workspace. Please fix this